### DISCUSSION TITLE
Can we add collaborator to live course in 5.0 version

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#725] [General Discussion]
Hello,
I am using Sunbird version 5.0
The issue I am facing is that when I am trying to add a collaborator to an already published course, it gives a 403 forbidden error.
The functionality is like that or This is an error. please guide us in this regard.
here is the error log
`Getting this error
[2024-05-28T06:47:20.197] [ERROR] api - {"id":"d3364992-efaa-e2c5-7155-8d42e05e468c","msg":"Cannot update collaborators because con │
│ tent is not in Draft state","err":{"errCode":"ERR_CONTENT_COLLABORATORS_UPDATE_FAILED","errMsg":"Update collaborators failed","resp │
│ onseCode":"FORBIDDEN"},"additionalInfo":{"contentId":"do_11405909839872000011564","qs":{"mode":"edit","fields":"versionKey,collabor │
│ ators,contentType,name,mimeType,framework,status"}}}
`
Check the recording

https://github.com/Sunbird-Ed/Community/assets/109504832/6fd96516-ca20-4fbe-af1c-70590f52948c

@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by vinukumar-vs** [comment]
@ashokreddy1208 Can you please check this and respond to this thread.

**Comment by ashokreddy1208** [comment]
@sachinb80 We have a know bug in jira which is open from long time, please find ticket details here - https://project-sunbird.atlassian.net/browse/SB-21971
cc @vinukumar-vs @mohitga

**Reply by sachinb80** [reply]
Ok @ashokreddy1208
